pythonfilemover/main.py

```diff
@@ -212,9 +217,53 @@ def process_file_move(self):
         shutil.move(self.directory_from, self.directory_to)
         print(f"SUCCESS: The Folder has been moved from {self.directory_from} to {self.directory_to}")
 
+class MoveFilesToTrashCommand():
+    def __init__(self, dir_a, dir_b): 
+        self.directory_from = None
```

Comment:
I am not understanding setting self.directory_from to None.  The user would need to define a from directory as we want to move all files from self.directory_from to the recycle bin.